SDKs/PowerShell/JumpCloud.SDK.V2/custom/generated/Set-JcSdkSystemAssociation.ps1

```diff
@@ -48,13 +48,22 @@ curl -X POST https://console.jumpcloud.com/api/v2/systems/{System_ID}/associatio
   }'
 ```
 .Example
-PS C:\> {{ Add code here }}
+PS C:\> Set-JcSdkSystemAssociation -SystemId <SystemID> -id <UserID> -op 'add' -type 'user' -Attributes @{ 'sudoEnabled' = $true; 'SudoWithoutPassword' = $true }
 
-{{ Add output here }}
 .Example
-PS C:\> {{ Add code here }}
+PS C:\> Set-JcSdkSystemAssociation -SystemId <SystemID> -id <UserID> -op 'update' -type 'user' -Attributes @{ 'sudoEnabled' = $true; 'SudoWithoutPassword' = $true }
 
-{{ Add output here }}
+.Example
+PS C:\> Set-JcSdkSystemAssociation -SystemId <SystemID> -id <UserID> -op 'remove' -type 'user' -Attributes @{ 'sudoEnabled' = $false; 'SudoWithoutPassword' = $false }
```

Comment:
One thing some customers might like better is just. It does the same thing, that's the whole bit we added to the build custom functions

`Set-JcSdkSystemAssociation -SystemId <SystemID> -id <UserID> -op 'remove' -type 'user'`